src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -1103,21 +1119,64 @@ private boolean addUserEmail(String siteId, String userRole) {
 		if(!errorMessage.isEmpty()) {
 			rv.put(Utils.PARAM_ERROR_MESSAGE, errorMessage);
 		}
-		
+
 		return rv;
 	}
-	
+
+	// Choose how to process each member.
+	// In java 8 could just assign the method to a variable early on and not need to keep redundent checking.
+	private void processEachMemberInSite(HashMap<String, Object> rv, JSONArray members, int iMember, String... format) {
+		{
+			if ("short".equals(format[0])) {
+				formatShortMemberInfo(rv, members, iMember);
+			}
+			if ("long".equals(format[0])) {
```

Comment:
`short` and `long` should be made as constants as these are referenced more than one place